tests/Http/PostTest.php

```diff
@@ -159,6 +159,75 @@ public function testCreatingAPost()
         ]);
     }
 
+    /**
+     * Test that a POST request to /posts creates a new text post.
+     *
+     * @return void
+     */
+    public function testCreatingATextPost()
+    {
+        $signup = factory(Signup::class)->create();
+        $quantity = $this->faker->numberBetween(10, 1000);
+        $text = $this->faker->sentence;
+        $details = ['source-detail' => 'broadcast-123', 'other' => 'other'];
+
+        // Mock the Blink API call.
+        $this->mock(Blink::class)->shouldReceive('userSignupPost');
+
+        // Create the post!
+        $response = $this->withAccessToken($signup->northstar_id)->postJson('api/v3/posts', [
+            'northstar_id'     => $signup->northstar_id,
+            'campaign_id'      => $signup->campaign_id,
+            'campaign_run_id'  => $signup->campaign_run_id,
+            'type'             => 'text',
+            'action'           => 'test-action',
+            'quantity'         => $quantity,
```

Comment:
Do we use think we'll ever use quantities with text posts? I know You've Got The Power is a quantity-less text submission, so if that's the template we follow moving forward it might be something to omit in our test case.